src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -506,4 +506,456 @@ select
+                      else 1 end as sped
+           ,case when e.lep_status = true then 1
+                 else 0 end as lep
+           ,lunch_application_status as lunch_status
+     from {{ ref("base_powerschool__student_enrollments") }} as e
+     where academic_year = 2022 and enroll_status = 0 and rn_year = 1 and grade_level <= 4
+),
+
+schedule as
+(
+      select c._dbt_source_relation
+            ,cast(c.cc_academic_year as string) as schedule_academic_year
+            ,'Kipp nJ/miami' as schedule_district
+            ,e.region as schedule_region
+            ,e.schedule_school_abbreviation
+            ,c.cc_schoolid as schedule_school_id
+            ,e.school_name as schedule_school
+            ,c.cc_studentid as student_id_schedule
+            ,e.student_number_enrollment as student_number_schedule --needed to keep track of students who test at a grade level different from the class they are enrolled at
+            ,case when c.courses_course_name in ('ela grK','ela K') then 'K'
+                  when c.courses_course_name = 'ela gr1' then '1'
+                  when c.courses_course_name = 'ela gr2' then '2'
+                  when c.courses_course_name = 'ela gr3' then '3'
+                  when c.courses_course_name = 'ela gr4' then '4'
+                  end as schedule_student_grade_level
+            ,c.cc_teacherid as teacher_id
+            ,c.teacher_lastfirst as teacher_number
+            ,c.teacher_lastfirst as teacher_name
+            ,c.courses_course_name as course_name
+            ,c.cc_course_number as course_number
+            ,c.cc_section_number as section_number
+
+            ,period as expected_test
+      from {{ ref("base_powerschool__course_enrollments") }} as c
+      left join --this join is needed to bring over the student_number onto the scheduling table, as mclass data contains only the student number field
+      (
+            select e._dbt_source_relation
+                  ,cast(e.academic_year as string) as academic_year
+                  ,e.region
+                  ,e.schoolid as school_id
+                  ,e.school_abbreviation as schedule_school_abbreviation
+                  ,e.school_name
+                  ,e.studentid as student_id
+                  ,e.student_number as student_number_enrollment
+                  ,e.enroll_status
+            from {{ ref("base_powerschool__student_enrollments") }} as e
+            where academic_year = 2022 and enroll_status = 0 and rn_year = 1 and grade_level <= 4
+      ) as e
+      on cast(c.cc_academic_year as string) = e.academic_year and c.cc_studentid = e.student_id and {{ union_dataset_join_clause(left_alias="c", right_alias="e") }}
+      cross join unnest({{ periods }}) as period
+      where c.cc_academic_year = 2022 and not c.is_dropped_course and not c.is_dropped_section and c.rn_course_number_year = 1 and c.courses_course_name in ('ELA grK','ELA K','ELA gr1','ELA gr2','ELA gr3','ELA gr4') and e.enroll_status = 0

```

Comment:
ensure the case is correct on the course names

Reply:
Good catch!